src/components/Integrations/BigQuery/AuthorizationPage.js

```diff
@@ -36,6 +36,8 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
     fetchData();
   }, [bigQueryClient, dopplerUserApiClient]);
 
+  const onReset = useCallback(async () => {}, []);
```

Comment:
What is it doing?

Reply:
Im forget remove this from code, the code was removed

Reply:
do you have doubts about when to use the `useCallback` hook ? @kapoc3